lib/cache/Cache.js

```diff
@@ -8,7 +8,7 @@ const { rimraf, exists, readdir, stat, rename } = require('./fsAsync')
 const fetchAndExtract = require('./fetchAndExtract')
 
 class Cache {
-    constructor({ name, cacheRoot }) {
+    constructor({ name, cacheRoot, requestOpts = {} }) {
```

Comment:
How's requestOpts stored? I don't see any `this.requestOpts = requestOpts`.

Reply:
Good catch, I apparantly missed this

Reply:
I was curious why `cli-style` didn't catch this unused variable, but it's because eslint was introduces in `cli-style@3.1.0` and we're still on `3.0.0` here.  I'll pin the `cli-style` version so greenkeeper keeps us up-to-date here.